pkg/build/builder/cmd/scmauth/scmauth.go

```diff
@@ -0,0 +1,25 @@
+package scmauth
```

Comment:
if we want to externalize it, then there is no point on baking it here. externalizing this from its own package living in 'pkg/util' will be much easier ;-) also the naming seems wrong to me.